app/api/v1/endpoints/postmark_webhook.py

```
@@ -119,11 +175,121 @@ async def test_webhook_processing(
         
         # Process using the same logic as main endpoint
-        return await process_postmark_webhook(mock_webhook, db, authenticated)
+        return await process_postmark_webhook(mock_webhook, db)
         
     except Exception as e:
-        logger.error(f"Error in test webhook processing: {str(e)}")
+        logger.error(f"💥 Error in test webhook processing: {str(e)}")
         raise HTTPException(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
             detail=f"Test processing failed: {str(e)}"
+        )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)